pages/services/edge-lb/1.0.0/upgrading/index.md

```diff
@@ -8,24 +8,46 @@ excerpt:
 enterprise: false
 ---
 
-Perform an Edge-LB upgrade by uninstalling your current version of Edge-LB, then reinstall from the remote repositories.
+Perform an Edge-LB upgrade by following this procedure:
```

Comment:
State what aspects of Edge-LB this would be disruptive to. Just control plane (API Server) or Pools, instances etc.